README.md

```diff
@@ -47,4 +47,4 @@ For example, if you download [scalabrad-0.5.0.tar.gz](https://bintray.com/artifa
 
 The scala manager is API-compatible with the old delphi manager, but it stores registry data in a different format because the old registry format was ill-defined and had problems encoding and decoding some labrad data. If you have existing registry data that was saved with the old manager, you can use the included migration tool to migrate the data to the new format. The migration tool is in the bin directory with the manager, e.g. `$PATH/scalabrad-0.5.0/bin/labrad-migrate-registry`. We do not currently provide a way to migrate data back to the old delphi format, so you should migrate when you are ready to switch to the scala manager exclusively.
 
-Use the `--help` flag for documentation of all available command line and environment variable configuration options. In particular, for v0.5.0 or above, if sessions without TLS authentication will connect to the manager, the manager must be called with the flag `--tls-required=false`.
+Use the `--help` flag for documentation of all available command line and environment variable configuration options. In particular, for v0.5.0 or above, if sessions without TLS authentication will connect to the manager, the manager must be called with the flag `--tls-required=false`. Note that environment variables must be in uppercase.
```

Comment:
I'd suggest a slight wording change: instead of "variables" put "variable names".